### PR TITLE
LPS-128990 Audit site vocabularies in the content audit tool

### PR DESCRIPTION
**Pull request by 4lejandrito**
**Motivation**
As the content audit tool has revealed itself to be a useful tool for content marketing teams, we aim to enlarge the scope of the feature by allowing to audit also the site vocabularies.

**Design**
https://www.figma.com/file/p6EWXgWGps5gtaLyMIiaFZ/LPS-128989-site-vocabularies-in-the-content-audit-tool?node-id=2%3A16573

**Requirements**
When opening the graph's configuration panel, the list of "Available" vocabularies will include all vocabularies from all the sites & asset libraries in the instance, along with the global ones
Vocabularies will show the name of the site or asset library they belong to
The selected vocabularies in the right box (Current) will determine which vocabularies are available for selection in the left box, so that the user cannot select two vocabularies from different sites (the user can always select a global vocab + a site vocab)
We will add a new sentence to the copy in the modal after "Select a maximum... "X" axis". Only vocabularies from the same site can be combined. Global vocabularies can be combined with vocabularies from any site.

### CODE REVIEW

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/edit_content_dashboard_configuration.jsp [hunk 1]
```diff
@@ -45,16 +50,22 @@ ContentDashboardAdminConfigurationDisplayContext contentDashboardAdminConfigurat
...
-						rightBoxMaxItems="<%= 2 %>"
-						rightBoxName="currentAssetVocabularyNames"
-						rightList="<%= contentDashboardAdminConfigurationDisplayContext.getCurrentVocabularyNames() %>"
-						rightReorder="<%= Boolean.TRUE.toString() %>"
-						rightTitle="in-use"
-					/>
+					<div>
+						<react:component
+							module="js/VocabulariesSelectionBox"
+							props='<%=
```
[the remark this answers is not in the document]
Comment (claraizquierdo) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/edit_content_dashboard_configuration.jsp:56]: I think that @YurenaC or @4lejandrito know more about this than me.

**Comment by YurenaC** [comment]
ci:sf

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by YurenaC** [comment]
ci:stop

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by YurenaC** [comment]
ci:forward

**Pull request closed without merging**